src/main/java/com/RaiseTech8/MyBatisdemo/NameController.java

```diff
@@ -0,0 +1,21 @@
+package com.RaiseTech8.MyBatisdemo;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+public class NameController {
+    private NameMapper nameMapper;
+
+    public NameController(NameMapper nameMapper) {
+        this.nameMapper = nameMapper;
+    }
+
+    @GetMapping("/national_flower")
+    public List<Name> national_flower() {
+        List<Name> national_flower = nameMapper.findAll();
```

Comment:
Javaの変数名は小文字始まりのキャメルケースです。
```suggestion
        List<Name> nationalFlower = nameMapper.findAll();
```

Reply:
c2fb0f4e11dcc4409998751c914c49e4b9277329

- 変数名をキャメルケース表記に変更しました。